test_scripts/Defects/5_0/2446_SDL_doesn_t_resume_HMILevel_when_DEACTIVATE_HMI_with_isActive_is_true_then_false.lua

```diff
@@ -28,6 +28,7 @@ config.application1.registerAppInterfaceParams.isMediaApplication = false
 -- [[ Local Functions ]]
 local function updateSDLfile()
   commonFunctions:write_parameter_to_smart_device_link_ini("ApplicationResumingTimeout", 5000)
+  commonFunctions:write_parameter_to_smart_device_link_ini("HMICapabilitiesCacheFile", "")
```

Comment:
@LitvinenkoIra I would propose to remove `user_modules/shared_testcases/commonFunctions` dedpendency from this script.
By the way this approach fixes that this script corrupts SDL ini file, because does not backup and restore it.

```
local common = require('user_modules/sequences/actions')
local utils = require("user_modules/utils")
local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
local test = require("user_modules/dummy_connecttest")

...

local function updateSDLfile()
  commonFunctions:write_parameter_to_smart_device_link_ini("ApplicationResumingTimeout", 5000)
  commonFunctions:write_parameter_to_smart_device_link_ini("HMICapabilitiesCacheFile", "")
end
```
=>
```
local common = require('user_modules/sequences/actions')
local utils = require("user_modules/utils")
local test = require("user_modules/dummy_connecttest")

...

local function updateSDLfile()
  common.sdl.setSDLIniParameter("ApplicationResumingTimeout", 5000)
  common.sdl.setSDLIniParameter("HMICapabilitiesCacheFile", "")
end
```

Reply:
@aderiabin Done in c473ab37f31c68da82457806ce360681f99c8db3